[2025-06-04T01:41:33.658+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-04T01:41:33.688+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ejemplo_bash_dag_s5.list_files scheduled__2025-06-03T00:00:00+00:00 [queued]>
[2025-06-04T01:41:33.701+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ejemplo_bash_dag_s5.list_files scheduled__2025-06-03T00:00:00+00:00 [queued]>
[2025-06-04T01:41:33.701+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 3
[2025-06-04T01:41:33.722+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): list_files> on 2025-06-03 00:00:00+00:00
[2025-06-04T01:41:33.736+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=12211) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-06-04T01:41:33.741+0000] {standard_task_runner.py:72} INFO - Started process 12212 to run task
[2025-06-04T01:41:33.738+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ejemplo_bash_dag_s5', 'list_files', 'scheduled__2025-06-03T00:00:00+00:00', '--job-id', '292', '--raw', '--subdir', 'DAGS_FOLDER/Sesion5/DAG_bash_s5.py', '--cfg-path', '/tmp/tmptuu5qoa7']
[2025-06-04T01:41:33.742+0000] {standard_task_runner.py:105} INFO - Job 292: Subtask list_files
[2025-06-04T01:41:33.756+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-04T01:41:33.785+0000] {task_command.py:467} INFO - Running <TaskInstance: ejemplo_bash_dag_s5.list_files scheduled__2025-06-03T00:00:00+00:00 [running]> on host 31d54648d280
[2025-06-04T01:41:33.787+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-04T01:41:33.860+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ejemplo_bash_dag_s5' AIRFLOW_CTX_TASK_ID='list_files' AIRFLOW_CTX_EXECUTION_DATE='2025-06-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-03T00:00:00+00:00'
[2025-06-04T01:41:33.860+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-04T01:41:33.877+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-04T01:41:33.878+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'ls -la /tmp/my_dir_datapath_s5']
[2025-06-04T01:41:33.890+0000] {subprocess.py:86} INFO - Output:
[2025-06-04T01:41:33.905+0000] {subprocess.py:93} INFO - total 8
[2025-06-04T01:41:33.906+0000] {subprocess.py:93} INFO - drwxrwxr-x 2 *** root 4096 Jun  4 01:41 .
[2025-06-04T01:41:33.906+0000] {subprocess.py:93} INFO - drwxrwxrwt 1 root    root 4096 Jun  4 01:41 ..
[2025-06-04T01:41:33.906+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-06-04T01:41:33.929+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-04T01:41:33.929+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ejemplo_bash_dag_s5, task_id=list_files, run_id=scheduled__2025-06-03T00:00:00+00:00, execution_date=20250603T000000, start_date=20250604T014133, end_date=20250604T014133
[2025-06-04T01:41:33.960+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-04T01:41:34.000+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-04T01:41:34.002+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
